tests/UI/pages/FO/checkout/index.js

```diff
@@ -223,6 +225,11 @@ class Checkout extends FOBasePage {
     return this.elementVisible(page, this.conditionToApproveCheckbox, 1000);
   }
 
+  async getTermsOfServicePageTitle(page) {
+    await page.click(this.termsOfServiceLink);
```

Comment:
If it's a navigation link, please use this instead

```suggestion
    await this.clickAndWaitForNavigation(page, this.termsOfServiceLink);
```

Reply:
No, it's not a navigation link, when we click a modal is displayed.

Reply:
ok thanks